lib/ts-types/monaco.d.ts

```diff
@@ -0,0 +1,5943 @@
+/*!-----------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Type definitions for monaco-editor v0.16.1
+ * Released under the MIT license
+*-----------------------------------------------------------*/
```

Comment:
Again, probably should not be copied in this repository, but pulled during the build.